presentation/backend/src/presentation-backend/SelectionScopesHelper.ts

```diff
@@ -272,3 +239,17 @@ const skipTransients = (callback: (id: Id64String) => void) => {
       callback(id);
   };
 };
+
+interface ComputeElementSelectionScopeProps {
+  iModel: IModelDb;
+  elementIds: Id64String[];
+  level: number;
+}

```

Comment:
Misplaced interface definition.

Reply:
Not sure what you meant, but this is now gone.